### PR TITLE
Updated docs added explanation.

### PR DESCRIPTION
**Pull request by absolutelightning**
### Description

Updated docs added explanation for https://github.com/hashicorp/consul/pull/17582#issuecomment-1591660623

### CODE REVIEW

File: website/content/commands/operator/raft.mdx [hunk 1]
```diff
@@ -66,6 +66,10 @@ Raft configuration.
 `Voter` is "true" or "false", indicating if the server has a vote in the Raft
 configuration.
 
+`Commit Index` is the last log index the server has a record of in its Raft log.
+
+`Trails Leader By` is the difference in commit index the server has from the leader.
```
Comment (analogue) [inline website/content/commands/operator/raft.mdx:71]:
```suggestion
`Trails Leader By` is the number of commits a follower trails the leader by.
```

**Review by analogue — APPROVED** [review]
Fix and 🚀 .

**Comment by david-yu** [comment]
@absolutelightning This also needs to be back ported to release branches where the associated PR would make it in. Thanks for the docs change!

File: website/content/commands/operator/raft.mdx (hunk 1 above)
Comment (absolutelightning) [reply website/content/commands/operator/raft.mdx:71]: I had enabled auto merge - please approve - https://github.com/hashicorp/consul/pull/17763

**Pull request merged**